src/08-lazy-evaluation/lazy-evaluation.md

```diff
@@ -4,8 +4,8 @@ The content of this chapter is available as a scala file [here.](./lazy-evaluati
 
 
 
-// load 07-fae.scala before loading this file or copy&paste it here 
-```scala mdoc:invisible
+// load 07-fae.scala before loading this file or copy&paste it here
+```scala mdoc:silent
```

Comment:
I would leave that as invisible here. I think thereis no need to show this code in the lecture notes.